lib/std/Build.zig

```diff
@@ -2085,8 +2087,49 @@ pub fn run(b: *Build, argv: []const []const u8) []u8 {
     };
 }
 
+pub const SearchPrefix = union(enum) {
+    /// Try to find `bin`, `lib`, and `include` sub-dirs.
+    all: LazyPath,
+    /// Use as written, without guessing sub-dirs.
+    /// If some path is not passed, it will be skipped.
+    exact: SearchPrefix.Exact,
+
+    pub const Exact = struct {
+        binaries: ?LazyPath = null,
+        libraries: ?LazyPath = null,
+        includes: ?LazyPath = null,
+    };
```

Comment:
```suggestion
pub const SearchMethod = union(enum) {
    /// Try to find `bin`, `lib`, and `include` sub-dirs.
    prefix: LazyPath,
    /// Use as written, without guessing sub-dirs.
    /// If some path is not passed, it will be skipped.
    paths: Paths,

    pub const Paths = struct {
        binaries: ?LazyPath = null,
        libraries: ?LazyPath = null,
        includes: ?LazyPath = null,
    };
```

Reply:
done